docs/ff-concepts/localization-accessibility/localization.md

```diff
@@ -0,0 +1,221 @@
+
+### Inside Properties Panel
+
+You can also add or update translations for individual text directly inside the properties panel. To do so, select the widget (e.g., Text, TextField, etc.), go to the properties panel, and click on the Globe icon. This will open a new panel.
+
+- To manually add or update a translation, make changes directly in the box under the language name.
+- To auto-translate for all languages, click on **Google Translate**.
+
+![manage-translation-in-properties-panel.avif](imgs/manage-translation-in-properties-panel.avif)
+
+## Translating Predefined Messages
+
+FlutterFlow allows you to manage the translation for the following types of predefined messages.
+
+- **iOS Permission Messages**: iOS permission messages are the prompts shown to iOS users when your app requests access to device features, such as the camera or photo library.
+- **Preset In-App Messages**: These are built-in messages that FlutterFlow displays for specific actions, such as authentication and file upload actions.
+
+To add translations for predefined messages, navigate to **Settings and Integrations** > **Project Setup** > **Languages**. Scroll down to the **Translation** section and select the category containing the message you wish to translate.
+
+Start by entering your message in the base language. Then, either use the **Translate Message** button for automatic translation or manually add your translations and mark them as **Fixed** to prevent them from being overridden by auto-translate.
+
+:::info
+
+Permission messages are displayed based on the features included in your app. For instance, Camera and Photo Library permission messages appear when a page contains a button with the **Upload Photo/Video action**.
+
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/tTmqFpnDLdCgVXHl9kDz?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+## Retrieving Language Related Data
+
+When building a multi-language app, you may need data like the current language code or language-specific text.
+
+In FlutterFlow, you can retrieve the following types of language-related data:
+
+- **Current Language Code**: This provides the ISO language code for the current app language (e.g., en, de, fr).
+- **Language-Dependent Text**: Allows you to specify different values for each language. For instance, you might want to display a country flag or name based on the current app language.
```

Comment:
I havent used this much, so dont know much about this language dependent text. Who is specifying these values for each language? the user? From the image it seems to me that I can use this value on text properties but then I dont get "Allows you to specify different values for each language"

Reply:
Yes, it is set by the user. Think of it like conditional values based on language. It is helpful in cases where you don't want to show translated text instead, show something else like a flag or anything.